plugins/obs-ffmpeg/obs-ffmpeg-source.c

```diff
@@ -452,6 +463,9 @@ static void ffmpeg_source_update(void *data, obs_data_t *settings)
 	s->speed_percent = (int)obs_data_get_int(settings, "speed_percent");
 	s->is_local_file = is_local_file;
 	s->seekable = obs_data_get_bool(settings, "seekable");
+	ffmpeg_options =
+		(char *)obs_data_get_string(settings, "ffmpeg_options");
```

Comment:
This cast seems wrong, it would be better to make `char *ffmpeg_options` a `const char *ffmpeg_options` instead. I see the existing code is also wrong here, I will fix that in a separate commit.